package.json

```diff
@@ -14,10 +14,10 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.12.13",
-    "@bcrs-shared-components/corp-type-module": "1.0.12",
-    "@bcrs-shared-components/enums": "1.0.40",
+    "@bcrs-shared-components/corp-type-module": "1.0.13",
+    "@bcrs-shared-components/enums": "1.0.41",
     "@bcrs-shared-components/genesys-web-message": "1.0.0",
-    "@bcrs-shared-components/interfaces": "1.0.22",
+    "@bcrs-shared-components/interfaces": "1.0.65",
```

Comment:
Updating enums and interfaces _should_ be safe since we almost always add to them instead of remove things. Plus, this is a feature branch, so even if this breaks something then at least it's a step ahead and we can fix it.